config/prod/JKG-Scicomp-Midi.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v10.j2
+stack_name: JKG-Scicomp-Midi2
```

Comment:
could you please make this match the filename or vise versa?

Reply:
Should be good to go. Also fixed yaml file extension